.github/workflows/test-osbuild-composer-integration.yml

```diff
@@ -53,6 +53,10 @@ jobs:
     runs-on: ubuntu-latest
     container:
       image: registry.fedoraproject.org/fedora:latest
+    env:
+      # workaround for expired cert at source of indirect dependency
+      # (go.opencensus.io/trace)
+      GOPROXY: "https://proxy.golang.org|direct"
```

Comment:
I'd document as making the downloading more robust - as opencensus was only the trigger but this should stay…
Also I'd move it to the Makefile to implement this only once and also benefit from it locally (when running on any fedora spin)
I'll suggest a  PR for this

Reply:
I can also do the PR after this is merged 🤷

Reply:
The Makefile setup is a bit annoying.  The whole container build mean to replicate the GH setup obfuscates more than it helps IMO.  I much prefer the old setup where things are visible in the action and there are fewer layers between the action config, the logs, and what's actually being run.

Also, if the Makefile is meant to be for people that clone the repo and want to run the tests without understanding all the details, I think it should also not make many assumptions about those people's setup.  Overriding their (potentially system-wide, sometimes necessary) GOPROXY seems a bit invasive and can cause more problems than it solves.

Reply:
I think setting GOPROXY with `?=` leaving people to just override if needed could be ok.
Also I'm not sure if duplicating `GOPROXY` definition 7 times is better than one indirection to make 🤔

Reply:
For build stability, I'd definitely go for keeping it. It says in the documentation the `|` pipe can be used to also cover timeouts…
but we can do this in a followup PR for sure…